options/options.c

```diff
@@ -194,6 +194,7 @@ static const m_option_t mp_vo_opt_list[] = {
     {"x11-wid-title", OPT_BOOL(x11_wid_title)},
 #endif
 #if HAVE_WAYLAND
+    {"wayland-colorspace-hint", OPT_BOOL(wl_colorspace_hint)},
```

Comment:
Do we need another variable for that? Isn't `target-colorspace-hint` enough?

Reply:
I'm not really sure. I think it depends on if forwarding metadata to the compositor is always wanted or not.

Reply:
Ok, I don't mind having an option, but it is getting a lot for simple colorspace signaling ;p